cmd/src/bin/tikv-ctl.rs

```diff
@@ -76,7 +76,7 @@ fn new_debug_executor(
 
             let raft_path = raft_db
                 .map(ToString::to_string)
-                .unwrap_or_else(|| format!("{}/../raft", kv_path));
+                .unwrap_or_else(|| format!("{}/raft", &cfg.storage.data_dir));
```

Comment:
This can't handle `tikv-ctl --db path/to/db` correctly.

Reply:
Yes, it's expected, we cannot find raft path without base-dir

Reply:
It's not expected. In most cases, "../raft" is a good guess. Changing it can break existing workflow and docs.